src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import {Dimensions, StyleSheet, Text, View} from 'react-native';
+import FastImage from 'react-native-fast-image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `FastImage` from 'react-native-fast-image' is a new addition. Please ensure that the package 'react-native-fast-image' has been added to your package.json and installed correctly. If not, it could lead to runtime errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->